navmodel-samples/src/main/kotlin/com/bumble/appyx/navmodel/cards/operation/CardsOperation.kt

```diff
@@ -0,0 +1,6 @@
+package com.bumble.appyx.navmodel.cards.operation
+
+import com.bumble.appyx.core.navigation.Operation
+import com.bumble.appyx.navmodel.cards.Cards
+
+sealed interface CardsOperation<T> : Operation<T, Cards.State>
```

Comment:
sealed? I think we agreed on removing it and allowing clients define custom operations